site/en/blog/media-updates-in-chrome-62/index.md

```diff
@@ -0,0 +1,216 @@
+MSE events.
+
+[Intent to Ship](https://groups.google.com/a/chromium.org/d/msg/blink-dev/-LTXhyDzS_E/LfjqN71kAAAJ) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5671401352593408) &#124;
+[Chromium Bug](https://crbug.com/623698)
+
+## FLAC in MP4 for MSE {: #flac-in-mp4-with-mse }
+
+The lossless audio coding format [FLAC] has been supported in regular media
+playback since Chrome 56. FLAC in ISO-BMFF support (aka FLAC in MP4) was added
+shortly after. And now FLAC in MP4 is available in Chrome 62 for [Media Source
+Extensions (MSE)].
+
+For info, Firefox folks are the ones who developed and implemented support for
+a [FLAC in MP4 encapsulation spec], and the BBC has been experimenting with
+using that with MSE. You can read the BBC's ["Delivering Radio 3 Concert
+Sound"] post to learn more.
+
+Here's how you can detect if FLAC in MP4 is supported for MSE:
+
+```js
+if (MediaSource.isTypeSupported('audio/mp4; codecs="flac"')) {
+  // TODO: Fetch data and feed it to a media source.
+}
+```
+
+If you want to see a full example, check out [our official sample].
+
+[Intent to Ship](https://groups.google.com/a/chromium.org/d/msg/blink-dev/ntoLfR7rbmE/3R1DQoBSAAAJ) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5713014258925568) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=666000)
+
+## Automatic video goes to fullscreen when the device is rotated {: #auto-fullscreen-rotate }
+
+If you rotate a device to landscape while a video is playing in the viewport,
+playback will automatically switch to fullscreen mode. Rotating the device to
+portrait puts the video back to windowed mode. Check out our [past article] for
+more details.
+
+<!-- lint disable definition-case -->
+
+[Encrypted Media Extensions (EME)]: https://w3c.github.io/encrypted-media/
+[Widevine Security Level 1]: https://web.archive.org/web/20180122175750/https://storage.googleapis.com/wvdocs/Widevine_DRM_Architecture_Overview.pdf
+[Sample Media PWA]: https://github.com/GoogleChrome/sample-media-pwa
+[https://biograf-155113.appspot.com/ttt/episode-2/]: https://biograf-155113.appspot.com/ttt/episode-2/
+[Shaka Player]: https://github.com/google/shaka-player
+[https://shaka-player-demo.appspot.com/demo/]: https://shaka-player-demo.appspot.com/demo/
+[Incognito mode]: https://support.google.com/chrome/answer/7440301?co=GENIE.Platform%3DAndroid
+[previous article]: /blog/media-updates-in-chrome-61#background-video-track-optimizations
+[the official sample]: https://googlechrome.github.io/samples/media/live-seekable-range.html
+[FLAC]: https://xiph.org/flac/
+[Media Source Extensions (MSE)]: https://developers.google.com/web/fundamentals/media/mse/basics
+[FLAC in MP4 encapsulation spec]: https://github.com/xiph/flac/blob/master/doc/isoflac.txt
+["Delivering Radio 3 Concert Sound"]: http://www.bbc.co.uk/rd/blog/2017-04-radio-3-high-quality-flac-dash
```

Comment:
here too, remove http:// and replace with https://

Reply:
Done